auth_saml/views/auth_saml.xml

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" ?>
+<odoo>
+    <!-- login form button -->
+    <template id="auth_saml.providers" name="Auth SAML Providers">
+        <t t-if="len(saml_providers) &gt; 0">
+            <em
+                t-attf-class="d-block text-center text-muted small my-#{len(saml_providers) if len(saml_providers) &lt; 3 else 3}"
+            >- or -</em>
+            <div class="o_login_saml2 list-group mt-1 mb-1 text-left">
+                <a
+                    t-foreach="saml_providers"
+                    t-as="p"
+                    class="list-group-item list-group-item-action py-2"
+                    t-att-href="p['auth_link']"
+                >
+                    <i t-att-class="p['css_class']" />
+                    <t t-esc="p['body']" />
+                </a>
+            </div>
+        </t>
+    </template>
+    <template id="auth_saml.login" inherit_id="web.login" name="Samlv2 Login buttons">
+        <xpath expr="//form" position="before">
+            <t t-set="form_small" t-value="True" t-if="len(saml_providers) &gt; 2" />
```

Comment:
```suggestion
            <t t-set="form_small" t-value="bool(saml_providers)" />
```

Reply:
The same code is present in auth_oauth.

It looks like your change would apply when there is 0 to 2 saml providers, which I do not believe should be done.

I’ve tried to see what is changed when there is 3 or more oauth providers or 3 or more SAML providers and can’t really see a visual difference.

I do not think this should be changed, to keep it consistent with auth_oauth.

Reply:
You're right, my suggestion was wrong, I meant `t-value="len(...) > 2"`.

Reply:
I’ve changed as requested.